README.md

```diff
@@ -0,0 +1,3 @@
+# Appsembler's `.github` repo
+
+Shared templates, defaults and configurations for GitHub repos.
```

Comment:
My understanding is that the README in a a `.github` project on Github will show up as the organization's page. Ie, on here: https://github.com/appsembler it will replace the "We make here Tahoe and Virtual Labs." blurb. I don't think we want Appsembler's main description to be "Shared templates, defaults and configurations for GitHub repos."

Reply:
Ah. nevermind. it's `profile/README.md` that does that.